cmd/clusterctl/cmd/delete_cluster.go

```diff
@@ -44,14 +46,16 @@ var deleteClusterCmd = &cobra.Command{
 	Long:  `Delete a kubernetes cluster with one command`,
 	Run: func(cmd *cobra.Command, args []string) {
 		if do.KubeconfigPath == "" {
-			exitWithHelp(cmd, "Please provide kubeconfig file for cluster to delete.")
+			exitWithHelp(cmd, "Please provide kubeconfig file for cluster to delete.\n")
```

Comment:
I added these newlines so that the output would be easier to read.

```
$ ./clusterctl delete cluster
Please provide kubeconfig file for cluster to delete.

NOTICE: clusterctl has been deprecated in v1alpha2 and will be removed in a future version.
Delete a kubernetes cluster with one command
...
```